### PR TITLE
INFRA-8255 Add fluentd-shipper chart

### PR DESCRIPTION
**Pull request by epicfilemcnulty**

### CODE REVIEW

File: stable/fluentd-shipper/values.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+image: "anchorfree/fluentd:v3.0"
+env:
+- name: FLUENTD_FORWARD_HOST
+  value: "fluentd.elasticsearch.afdevops.com"
```
Comment (zunkree) [inline stable/fluentd-shipper/values.yaml]: That address is obsolete, could you please consult with @pavdmyt about new dns name

Comment (pavdmyt) [reply stable/fluentd-shipper/values.yaml]: Please use `input.efk.afdevops.com` instead.

File: stable/fluentd-shipper/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+appVersion: 1.0.3
```
Comment (pavdmyt) [inline stable/fluentd-shipper/Chart.yaml]: Why *v1.0.3* if docker image is *v3.0*: `anchorfree/fluentd:v3.0`.

File: stable/fluentd-shipper/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+apiVersion: v1
+name: fluentd-shipper
+description: Fluentd log shipper
+home: https://github.com/AnchorFree/docker-fluentd
+version: 1.0.3
```
Comment (pavdmyt) [inline stable/fluentd-shipper/Chart.yaml]: Why *v1.0.3* for the newly added chart?

**Comment by epicfilemcnulty** [comment]
Thanks for the input, guys, but, as discussed with @zunkree, probably this chart should not be in the public. I'll close this one and update the chart in k8s-shared according to your advice.

**Pull request closed without merging**